cli/foundation-models/system/inference/text-to-image/text-to-image-batch-endpoint.sh

```diff
@@ -0,0 +1,92 @@
+set -x
+# the commands in this file map to steps in this notebook: https://aka.ms/azureml-infer-batch-sdk-text-classification
+
+# script inputs
+registry_name="azureml"
+subscription_id="<SUBSCRIPTION_ID>"
+resource_group_name="<RESOURCE_GROUP>"
+workspace_name="<WORKSPACE_NAME>"
+
+# This is the model from system registry that needs to be deployed
+model_name="runwayml-stable-diffusion-v1-5"
+
+# Path to input .csv file
+inputs_dir="./batch_data"
+
+# Validate the existence of the model in the registry and get the latest version
+model_list=$(az ml model list --name ${model_name} --registry-name ${registry_name} 2>&1)
+if [[ ${model_list} == *"[]"* ]]; then
+    echo "Model doesn't exist in registry. Check the model list and try again."; exit 1;
+fi
+version_temp=${model_list#*\"version\": \"}
```

Comment:
Use this.

https://github.com/Azure/azureml-examples/blob/19eb57e98a650cdd9530a1b9b7a31f94196539cc/cli/foundation-models/system/inference/image-object-detection/image-object-detection-batch-endpoint.sh#L51